### PR TITLE
Add missing files to qrc

### PR DESCRIPTION
**Pull request by naezith**

### CODE REVIEW

File: qml.qrc [hunk 1]
```diff
@@ -7,7 +7,13 @@
         <file>atomic_qt_design/assets/languages/atomic_qt_tr.qm</file>
         <file>atomic_qt_design/assets/languages/atomic_qt_fr.qm</file>
         <file>atomic_qt_design/assets/fonts/Rubik-Regular.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-SemiBold.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Medium.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Light.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Thin.ttf</file>
         <file>atomic_qt_design/assets/fonts/Montserrat-Regular.ttf</file>
+        <file>atomic_qt_design/assets/images/arrow_up.svg</file>
+        <file>atomic_qt_design/assets/images/arrow_down.svg</file>
```
Comment (tonymorony) [inline qml.qrc:16]: `arrow-down.svg` actual file name

File: qml.qrc [hunk 2]
```diff
@@ -7,7 +7,13 @@
         <file>atomic_qt_design/assets/languages/atomic_qt_tr.qm</file>
         <file>atomic_qt_design/assets/languages/atomic_qt_fr.qm</file>
         <file>atomic_qt_design/assets/fonts/Rubik-Regular.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-SemiBold.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Medium.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Light.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Thin.ttf</file>
         <file>atomic_qt_design/assets/fonts/Montserrat-Regular.ttf</file>
+        <file>atomic_qt_design/assets/images/arrow_up.svg</file>
```
Comment (tonymorony) [inline qml.qrc:15]: `arrow-up.svg` actual file name

File: qml.qrc [hunk 3]
```diff
@@ -7,7 +7,13 @@
         <file>atomic_qt_design/assets/languages/atomic_qt_tr.qm</file>
         <file>atomic_qt_design/assets/languages/atomic_qt_fr.qm</file>
         <file>atomic_qt_design/assets/fonts/Rubik-Regular.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-SemiBold.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Medium.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Light.ttf</file>
+        <file>atomic_qt_design/assets/fonts/Montserrat-Thin.ttf</file>
```
Comment (tonymorony) [inline qml.qrc:13]: there is no such files in fonts folder

**Comment by naezith** [comment]
@Milerius told me to add these. All these changes are for the new UI. I don't know how he is having the errors at `dev` branch. There are no typos or anything. Closing this then.

**Pull request closed without merging**